samples/net/common/common.cmake

```diff
@@ -1,14 +1 @@
 # Common routines used in net samples
-
-if(CONFIG_NET_TESTING)
```

Comment:
Would like to hear comment from @jukkar here. Maybe we shouldn't remove this chunk, but rename CONFIG_NET_TESTING.

Reply:
I managed to find the commit in 2016 that removed this from Kconfig. It really looks like we do not need this setting any more.